samples/subsys/logging/logger/README.rst

```diff
@@ -0,0 +1,71 @@
+.. _logger_sample:
+
+Logging
+###########
+
+Overview
+********
+A simple application that demonstrates use of logging subsystem. It demonstrates
+main features: severity levels, timestamping, module level filtering and 
+instance level filtering. It also showcases logging capabilities in terms of 
+performance.
+
+Building and Running
+********************
+
+This project outputs multiple log message to the console.  It can be built and
+executed on QEMU as follows:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/subsys/logging/logger
+   :host-os: unix
+   :board: qemu_x86
+   :goals: run
+   :compact:
+
+
+Sample Output
+=============
+
+.. code-block:: console
+
+         Module logging showcase.
```

Comment:
The body text for the code-block directive should only be indented by 3 spaces (aligned with the ``c`` in ``code-block``

Reply:
Done